Jenkinsfile

```diff
@@ -65,8 +65,12 @@ pipeline {
       steps {
         xvnc(useXauthority: true) {
           sh """
-            ./full-build.sh --tp=${selectedTargetPlatform()} \
-              ${javaVersion() == 17 ? '' : '--toolchains releng/toolchains.xml -Pstrict-release-jdk'}
+            mvn -f org.eclipse.xtext.full.releng --batch-mode clean deploy \
+              ${javaVersion() == 17 ? '' : '--toolchains releng/toolchains.xml -Pstrict-release-jdk'} \
+              -PuseJenkinsSnapshots \
+              -Dtarget-platform-classifier=xtext-${selectedTargetPlatform()} \
+              -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn \
+              -DaltDeploymentRepository=local::file:./build/maven-repository
```

Comment:
In the script the value was `-DaltDeploymentRepository=local::default::file:./build/maven-repository`, but Maven emitted a deprecation warning, therefore I changed that to the recommended value.